lib/geo.js

```diff
@@ -38,11 +38,11 @@ ReqlGeometry.prototype.distance = function(to, options) {
 
   if (to.type !== 'Point') {
     //TODO Implement
-    util.notAvailable('You can compute the distance to a point only in Reqlite (for now)', query)
+    util.notAvailable('You can compute the distance to a point only in Reqlite (for now)', /* ?? query */ {})
```

Comment:
is this right? there's no access to a `query` object in this scope, so I sent a regular object instead.